include/bluetooth/mesh/light_ctrl_reg.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ *  @defgroup bt_mesh_light_ctrl_reg Light Lightness Control Regulator
+ *  @ingroup bt_mesh_light_ctrl
+ *  @{
+ *  @brief Light Lightness Control regulator API
+ */
+
+#ifndef BT_MESH_LIGHT_CTRL_REG_H__
+#define BT_MESH_LIGHT_CTRL_REG_H__
+
+#include <bluetooth/mesh.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Light Ligthness Control regulator configuration. */
+struct bt_mesh_light_ctrl_reg_cfg {
+	/** Regulator upwards integral coefficient */
+	float kiu;
+	/** Regulator downwards integral coefficient */
+	float kid;
+	/** Regulator upwards proportional coefficient */
+	float kpu;
+	/** Regulator downwards proportional coefficient */
+	float kpd;
+	/** Regulator dead zone (in percent) */
+	float accuracy;
```

Comment:
Seems that would make sense only because we use this with the LC-server now. I'll keep it as float, that's what I'd done if I'd implemented this without knowledge of the spec.